packages/block-library/src/navigation/editor.scss

```diff
@@ -248,7 +252,12 @@ $color-control-label-height: 20px;
 		flex-wrap: nowrap;
 	}
 
-	// .. but hide entirely when the placeholder can still be toggled.
+	// Hide entirely when vertical.
+	.is-vertical.is-selected & {
+		display: none;
+	}
+
+	// Hide entirely when the placeholder can still be toggled.
 	.wp-block-navigation.is-selected .is-small & {
```

Comment:
When is this `is-small` class added? Having `display: none` on the placeholder when the block is selected, for either horizontal or vertical, doesn't seem to make any difference visually, so I must be missing something here 😅

Reply:
Thanks for reviewing. I can't see what this does either 🤔 

Maybe it can be removed.